sky/shell/platform/ios/FlutterView.h

```diff
@@ -5,10 +5,14 @@
 #ifndef FLUTTER_FLUTTERVIEW_H_
 #define FLUTTER_FLUTTERVIEW_H_
 
+#include "sky/services/semantics/semantics.mojom.h"
+
 #include <UIKit/UIKit.h>
 
 @interface FlutterView : UIView
 
+- (instancetype)initWithSemantics:(semantics::SemanticsServer*)semantics;
```

Comment:
The issue is how this approach scales once you're providing a large number of these sorts of objects to the view.  Should we keep adding to the init message until it has a large number of arguments?  It's simpler to make it a separate step.  That way future developers don't have to worry if there's some reason creating the bridge needs to be integrated with initializing the view.